package-lock.json

```diff
@@ -7,7 +7,6 @@
     "": {
       "name": "lavamoat-monorepo",
       "version": "0.0.0",
-      "hasInstallScript": true,
```

Comment:
i spotted this change too running `npm i` in `main`

think we forgot to commit this in
- https://github.com/LavaMoat/LavaMoat/pull/745

after removing `postinstall` and moving it to our `setup` script

Reply:
yeah.  I didn't realize it would need to be updated.  TIL!

Reply:
relevant: https://www.npmjs.com/package/dependency-cruiser